### ISSUE TITLE
Running BotFarm.exe

### ISSUE THREAD

**Issue by jjo72ddd** [jackpoz/BotFarm#55]
Hello!
Sorry if that sounds silly. But how to run this great program?
I downloaded from https://ci.appveyor.com/project/jackpoz/botfarm/build/artifacts. Filled all the settings in BotFarm.dll.config and tried to run the program. But it produced this error
![изображение](https://user-images.githubusercontent.com/24715026/81444186-6c2d3e80-917f-11ea-9c11-ea64ffe88a8a.png)

**Comment by jackpoz** [comment]
do you have .NET Core 3.1 installed ? https://dotnet.microsoft.com/download/dotnet-core/3.1

**Comment by jjo72ddd** [comment]
Yes, I installed it. (.NET Core 3.1 Runtime (v3.1.0) - Windows x64 Installer)

**Comment by jackpoz** [comment]
Do you have VS 2019 C++ runtime libraries installed too ?

**Comment by jjo72ddd** [comment]
I'm not sure that I have it installed, I don’t know where to download it. I downloaded

SDK 3.1.201
Visual Studio support
Visual Studio 2019 (v16.5)
Included runtimes
.NET Core Runtime 3.1.3
ASP.NET Core Runtime 3.1.3
Desktop Runtime 3.1.3

**Comment by jackpoz** [comment]
you could try building it then instead of using the appveyor binaries
